src/ports/postgres/modules/recursive_partitioning/decision_tree.py_in

```diff
@@ -669,6 +669,8 @@ def _get_n_and_deplist(training_table_name, dependent_variable, filter_null):
                    source=training_table_name,
                    filter_null=filter_null)
     r = plpy.execute(sql)[0]
+    if r['n_rows'] is None:
+        return None, None
```

Comment:
It's better to return `0` instead of the `None` for `n_rows`. The _assert in the calling function could be changed to `n_rows > 0`